internal/rsl/rsl.go

```diff
@@ -768,9 +768,5 @@ func isRelevantGittufRef(refName string) bool {
 		return false
 	}
 
-	if refName == gittufPolicyStagingRef {
```

Comment:
This is necessary, we return gittuf refs except for policy-staging, by design. We might want to rethink how this is filtered, but we can't just take this out right now.